perfkitbenchmarker/linux_benchmarks/tensorflow_benchmark.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run Tensorflow benchmarks (https://github.com/tensorflow/benchmarks)."""
+
+import logging
+import os
+import re
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import data
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker.linux_packages import cuda_toolkit_8
+
+FLAGS = flags.FLAGS
+
+CUDA_TOOLKIT_INSTALL_DIR = '/usr/local/cuda'
```

Comment:
Use `cuda_toolkit_8.CUDA_TOOLKIT_INSTALL_DIR` instead of redefining the constant.